commit 6cba00eaa95c1ace1fb511f534b233ae302ccb2e
Author: Per Forlin <per.forlin@linaro.org>
Date:   Tue Sep 13 23:03:28 2011 +0200

    fault-inject: export setup_fault_attr()
    
    mmc_core module needs to use setup_fault_attr() in order
    to set fault injection attributes during module load time.
    
    Signed-off-by: Per Forlin <per.forlin@linaro.org>
    Reviewed-by: Akinobu Mita <akinobu.mita@gmail.com>
    Signed-off-by: Chris Ball <cjb@laptop.org>

diff --git a/lib/fault-inject.c b/lib/fault-inject.c
index 328d43357ed2..4f7554025e30 100644
--- a/lib/fault-inject.c
+++ b/lib/fault-inject.c
@@ -14,7 +14,7 @@
  * setup_fault_attr() is a helper function for various __setup handlers, so it
  * returns 0 on error, because that is what __setup handlers do.
  */
-int __init setup_fault_attr(struct fault_attr *attr, char *str)
+int setup_fault_attr(struct fault_attr *attr, char *str)
 {
 	unsigned long probability;
 	unsigned long interval;
@@ -36,6 +36,7 @@ int __init setup_fault_attr(struct fault_attr *attr, char *str)
 
 	return 1;
 }
+EXPORT_SYMBOL_GPL(setup_fault_attr);
 
 static void fail_dump(struct fault_attr *attr)
 {